docker/dockerfile-build-ubuntu-16.04

```diff
@@ -14,7 +14,7 @@ ARG cmake_ver_minor=3.7.2
 ARG REPO_RADEON=repo.radeon.com
 
 # Install Packages
-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends curl && \
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends curl repo && \
```

Comment:
@david-salinas please refer to the latest `SYNC_UPSTREAM.md` . `repo` is an optional dependency now. I think we can remove it.